src/pdf.ts

```diff
@@ -627,7 +627,10 @@ namespace Private {
   export function ensurePDFJS(): Promise<any> {
```

Comment:
maybe make this `async` to clean up the promise chaining... not sure whether `Promise.all` is even going to buy us that much, anyhow.

Reply:
I cleaned it up a little bit: no more chaining inside of `ensurePDFJS`. Also it does not return the entire guts of pdfjs anymore, only `getDocument` and `PDFViewer` that are used downstream.